README.md

```diff
@@ -1,84 +1,93 @@
-        io:println(response["payload"]);
-    } else {
-        io:println(response.message());
-    }
-}
-```
-
-Follow the steps below to run this example.
-
-1. Save the example in a Ballerina file (e.g., `soapExample.bal`).
-2. Execute the `ballerina run soapExample.bal` command to run the file.
-You will get a response similar to the following.
-
-```xml
-<?xml version="1.0" encoding="UTF-8"?>
-<soap:Body xmlns:soap="http://www.w3.org/2003/05/soap-envelope">
-   <CheckPhoneNumberResponse xmlns="http://ws.cdyne.com/PhoneVerify/query">
-      <CheckPhoneNumberResult>
-         <Company>Toll Free</Company>
-         <Valid>true</Valid>
-         <Use>Assigned to a code holder for normal use.</Use>
-         <State>TF</State>
-         <RC />
-         <OCN />
-         <OriginalNumber>18006785432</OriginalNumber>
-         <CleanNumber>8006785432</CleanNumber>
-         <SwitchName />
-         <SwitchType />
-         <Country>United States</Country>
-         <CLLI />
-         <PrefixType>Landline</PrefixType>
-         <LATA />
-         <sms>Landline</sms>
-         <Email />
-         <AssignDate>Unknown</AssignDate>
-         <TelecomCity />
-         <TelecomCounty />
-         <TelecomState>TF</TelecomState>
-         <TelecomZip />
-         <TimeZone />
-         <Lat />
-         <Long />
-         <Wireless>false</Wireless>
-         <LRN />
-      </CheckPhoneNumberResult>
-   </CheckPhoneNumberResponse>
-</soap:Body>
-```
+Ballerina SOAP Library
+===================
+
+[![Build](https://github.com/ballerina-platform/module-ballerina-soap/actions/workflows/build-timestamped-master.yml/badge.svg)](https://github.com/ballerina-platform/module-ballerina-soap/actions/workflows/build-timestamped-master.yml)
+[![codecov](https://codecov.io/gh/ballerina-platform/module-ballerina-soap/branch/main/graph/badge.svg)](https://codecov.io/gh/ballerina-platform/module-ballerina-soap)
+[![Trivy](https://github.com/ballerina-platform/module-ballerina-soap/actions/workflows/trivy-scan.yml/badge.svg)](https://github.com/ballerina-platform/module-ballerina-soap/actions/workflows/trivy-scan.yml)
+[![GraalVM Check](https://github.com/ballerina-platform/module-ballerina-soap/actions/workflows/build-with-bal-test-graalvm.yml/badge.svg)](https://github.com/ballerina-platform/module-ballerina-soap/actions/workflows/build-with-bal-test-graalvm.yml)
+[![GitHub Last Commit](https://img.shields.io/github/last-commit/ballerina-platform/module-ballerina-soap.svg)](https://github.com/ballerina-platform/module-ballerina-soap/commits/main)
+[![Github issues](https://img.shields.io/github/issues/ballerina-platform/ballerina-standard-library/module/soap.svg?label=Open%20Issues)](https://github.com/ballerina-platform/ballerina-standard-library/labels/module%2Fyaml)
+[![codecov](https://codecov.io/gh/ballerina-platform/module-ballerina-soap/branch/main/graph/badge.svg)](https://codecov.io/gh/ballerina-platform/module-ballerina-soap)
```

Comment:
```suggestion
[![codecov](https://codecov.io/gh/ballerina-platform/module-ballerina-soap/branch/master/graph/badge.svg)](https://codecov.io/gh/ballerina-platform/module-ballerina-soap)
```
branches should change right?

Reply:
Fixed in [4ff4425](https://github.com/ballerina-platform/module-ballerina-soap/pull/89/commits/4ff44259f7f1db65553198c93f474b739234a811)